Code/Engine/Core/Interfaces/PhysicsWorldModule.h

```diff
@@ -168,6 +168,9 @@ class EZ_CORE_DLL ezPhysicsWorldModuleInterface : public ezWorldModule
 
   /// \brief Adds a fixed joint to pOwner.
   virtual void AddFixedJointComponent(ezGameObject* pOwner, const ezPhysicsWorldModuleInterface::FixedJointConfig& cfg) {}
+
+  /// \brief Gets world space bounds of a physics object if its shape type is included in shapeTypes and its collision layer interacts with uiCollisionLayer.
+  virtual ezBoundingBoxSphere GetWorldSpaceBounds(ezGameObject* pOwner, ezUInt32 uiCollisionLayer, ezBitflags<ezPhysicsShapeType> shapeTypes, bool bRecursive) const { return ezBoundingBoxSphere::MakeInvalid(); }
```

Comment:
Please add a short comment what recursive means here, ie that without it the bounds of child objects are ignored and.

Reply:
That's good enough I think.